.codespellignore

```diff
@@ -1,3 +1,9 @@
+# The following words are intentionally added to the ignore list because they are valid Meridian 59 codebase words.
```

Comment:
I was hoping for more specific comments on individual words.  For example, next to "Crystalize Mana", put a comment that this is a spell whose name is misspelled in the game.  For "asssign", say that this is a class for a sign for the assassin's game (I think?).  You don't need a general comment saying that Meridian 59 is a game that includes unique terms---you can assume the reader knows this.  Put yourself in the place of someone new to the codebase.  They come across "asssign" and they are confused as to why this is not a misspelling.

You don't necessarily need to comment on every single word, just the ones that are confusing.

Reply:
Understood, I refactored it based on your feedback.

I had to do comments on the following line, because when I included "#" after a word, it would not filter that word anymore.  Putting the comment on the following line seems to work.